migrations/versions/7eaa506f41dc_.py

```diff
@@ -0,0 +1,23 @@
+"""Add service_type column to node
+
+Revision ID: 7eaa506f41dc
+Revises: 56eff971505c
+Create Date: 2018-10-08 09:47:44.206268
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = '7eaa506f41dc'
+down_revision = '56eff971505c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('node', sa.Column('service_type', mysql.VARCHAR(length=255), server_default=sa.text("'openvpn'"), nullable=False))
```

Comment:
Default should be `''` or `NULL` - we dont know

Reply:
Basicly all the current service proposals do contain a service type, with openvpn being the value here. I've added a check for it on the register proposal endpoint, so in the future we'll not accept empty values. No reason to have it as nullable now, right?